sdk/identity/identity/src/credentials/azureCliCredential.ts

```diff
@@ -183,4 +180,35 @@ export class AzureCliCredential implements TokenCredential {
       }
     });
   }
+
+  /**
+   * Parses the raw JSON response from the Azure CLI into a usable AccessToken object
+   *
+   * @param rawResponse - The raw JSON response from the Azure CLI
+   * @returns An access token with the expiry time parsed from the raw response
+   *
+   * The expiryTime of the credential's access token, in milliseconds, is calculated as follows:
+   *
+   * When available, expires_on (introduced in Azure CLI v2.54.0) will be preferred. Otherwise falls back to expiresOn.
+   */
+  private parseRawResponse(rawResponse: string): AccessToken {
+    const response: any = JSON.parse(rawResponse);
+    const token = response.accessToken;
+    // if available, expires_on will be a number representing seconds since epoch.
+    // ensure it's a number or NaN
+    const expiresOnTimestamp = Number.parseInt(response.expires_on) * 1000;
```

Comment:
I remember it used to be important to always pass the radix to parseInt, but maybe that's not needed anymore?

Reply:
huh TIL 

```
If 0 or not provided, the radix will be inferred based on string's value. Be careful — this does not always default to 10! The [description below](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#description) explains in more detail what happens when radix is not provided.
```

I guess it could assume octal if it starts with 0 or something... I can add the radix!

I know about the pain of doing something like `["1", "2", "3"].map(Number.parseInt)` but figured in this case it would be fine. I'll be explicit just in case